console/src/components/IconButton/IconButton.js

```diff
@@ -0,0 +1,56 @@
+import AgateIcon from '@enact/agate/Icon';
+import Button from '@enact/agate/Button';
+import kind from '@enact/core/kind';
+import React from 'react';
+
+import componentCss from './IconButton.less';
+
+const Icon = kind({
+	name: 'Icon',
+	styles: {
+		css: componentCss,
+		className: 'icon'
+	},
+	computed: {
+		className: ({css, smallest, styler}) => {
+			return styler.append(smallest ? css.smallest : '');
```

Comment:
We could avoid the value-less empty string `''` fallback by using a modified expression:
```
className: ({css, smallest, styler}) => styler.append(smallest && css.smallest)
```
If `smallest` is set then we can simply return the css class, otherwise it evals to `false` and no styles are applied (and the same with `IconButton` below).